value/pom.xml

```diff
@@ -139,33 +143,34 @@
 	    </goals>
 	    <configuration>
 	      <minimizeJar>true</minimizeJar>
-              <filters>
-                <filter>
-                   <artifact>org.apache.velocity:*</artifact>
```

Comment:
It means that the filtered pom includes Velocity, so the Apache dependencies come in transitively, rather than being provided by the jar, so they participate in conflict resolution.